tests/integration/MyObservations.test.js

```diff
@@ -35,17 +39,15 @@ describe( "MyObservations", ( ) => {
   // } );
 
   describe( "when signed out", ( ) => {
-    async function testApiMethodNotCalled( apiMethod, language ) {
+    async function testApiMethodNotCalled( apiMethod ) {
       // Let's make sure the mock hasn't already been used
       expect( apiMethod ).not.toHaveBeenCalled( );
       const signedInUsers = global.realm.objects( "User" ).filtered( "signedIn == true" );
       expect( signedInUsers.length ).toEqual( 0 );
       renderAppWithComponent( <ObsList /> );
-      // TODO: We should really address this globally in the test suite. On first render,
-      // text doesn't have a language set, but on second render, text will default to English.
-      const textByLanguage = language === "en" ? "Log in to iNaturalist" : "Log-in-to-iNaturalist";
+      const loginText = i18n.t( "Log-in-to-iNaturalist" );
```

Comment:
That's been my preference, both to address flaky i18n init, but also because it should make it easier to detect *all* uses of a localized string, even in tests. That said, I don't think it needs to be mandatory, and if we can't express a preference like that as an eslint rule I'm not sure anyone's going to pay attention to it anyway. Better to just mention it in code reviews, IMO.